opentelemetry-api/src/opentelemetry/trace/propagation/__init__.py

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 from typing import Optional
 
-from opentelemetry.context import get_value, set_value
+from opentelemetry.context import _create_key, get_value, set_value
 from opentelemetry.context.context import Context
 from opentelemetry.trace.span import INVALID_SPAN, Span
 
-SPAN_KEY = "current-span"
```

Comment:
Is this not a breaking API change? cc @ocelotl

Reply:
Probably this was changed to avoid the public api test cases from breaking, `SPAN_KEY` should remain public and the tests will pass after the corresponding label is added. @eunice98k can you please fix this?

Reply:
It only checks for additions. The script is not meant to detect breaking changes, but to warn us when a possibly unnecessarily new public symbol is being added, to avoid our API from growing without need.